src/snowflake/cli/plugins/object/command_aliases.py

```diff
@@ -17,35 +19,62 @@ def add_object_command_aliases(
     app: SnowTyper,
     object_type: ObjectType,
     name_argument: typer.Argument,
-    like_option: typer.Option,
-    scope_option: typer.Option,
+    like_option: Optional[typer.Option],
+    scope_option: Optional[typer.Option],
+    ommit_commands: List[str] = [],
```

Comment:
typo here? ommit -> omitted

Reply:
Intended: "ommit these while creating aliases"

Reply:
It only contains 1 'm'.  ommit -> omit